snakes_cafe.py

```diff
@@ -0,0 +1,65 @@
+
+menu = """**************************************
+**    Welcome to the Snakes Cafe!   **
+**    Please see our menu below.    **
+**
+** To quit at any time, type "quit" **
+**************************************
+
+Appetizers
+----------
+Wings
+Cookies
+Spring Rolls
+
+Entrees
+-------
+Salmon
+Steak
+Meat Tornado
+A Literal Garden
+
+Desserts
+--------
+Ice Cream
+Cake
+Pie
+
+Drinks
+------
+Coffee
+Tea
+Blood of the Innocent
+
+***********************************
+** What would you like to order? **
+***********************************"""
+
+order = {
+    'Wings': 0,
+    'Cookies': 0,
+    'Spring Rolls': 0,
+    'Salmon': 0,
+    'Steak': 0,
+    'Meat Tornado': 0,
+    'A Literal Garden': 0,
+    'Ice Cream': 0,
+    'Cake': 0,
+    'Pie': 0,
+    'Coffee': 0,
+    'Tea': 0,
+    'Blood of the Innocent': 0
+}
+
+print(menu)
+
+order_line = input('> ')
+
+while order_line != 'quit':
+    if order_line in order:
+        order[order_line] += 1
+        print('You have added {} {} to your order'.format(order[order_line], order_line))
+        order_line = input('> ')
+    else:
+        print('Please enter a valid menu item')
+        order_line = input('> ')
```

Comment:
Since `order_line = input('> ')` is in both the `if` and `else` statements, it should be taken out of the conditional.